dask/dataframe/rolling.py

```diff
@@ -110,10 +113,15 @@ def _call_method(self, method_name, *args, **kwargs):
             self.rolling_kwargs, method_name, args, kwargs)}
         if self.rolling_kwargs['axis'] in [0, 'rows']:
             # roll in the partition direction (will need to access neighbor)
+            window = self.rolling_kwargs['window']
+            tail_name = '{}-tailprev-{}'.format(old_name, window-1)
```

Comment:
Most dask collections follow an informal convention of `descriptive-text-LONGHASHSTRING` which the diagnostic utilities (like the fancy progress bars) end up taking advantage of.  In particular it's nice for keynames to reduce nicely under the [key_split function](https://github.com/dask/distributed/blob/master/distributed/utils.py#L155-L200).  

So it would be nicer to have key names like

    rolling-tail-12341234

Rather than

    rolling-12341234-tail